data-prepper-plugins/s3-sink/src/test/java/org/opensearch/dataprepper/plugins/sink/S3SinkConfigTest.java

```diff
@@ -32,10 +34,17 @@ void default_max_upload_retries_test() throws NoSuchFieldException, IllegalAcces
     }
 
     @Test
-    void get_bucket_option_test() {
-        assertThat(new S3SinkConfig().getBucketOptions(), equalTo(null));
+    void get_bucket_name_test() {
+        assertThat(new S3SinkConfig().getBucketName(), equalTo(null));
```

Comment:
bucketName has a nonNull validation on it. Let's make this test be more realistic.